waspc/src/Generator/FileDraft/Writeable.hs

```diff
@@ -2,13 +2,14 @@ module Generator.FileDraft.Writeable
        ( Writeable(..)
        ) where
 
-import qualified Path.Aliases as Path
-
+import StrongPath (Path, Abs, Dir)
+import Generator.Common (ProjectRootDir)
 import Generator.FileDraft.WriteableMonad
 
+
 class Writeable w where
-    -- | Write file somewhere in the provided dst directory.
+    -- | Write file somewhere in the provided project root directory.
     write :: (WriteableMonad m)
-          => Path.AbsDir  -- ^ Absolute path of dst directory.
+          => Path Abs (Dir ProjectRootDir)
```

Comment:
maybe just add that dst part somewhere, e.g. `DstProjectRootDir`? or `GenProjectRootDir`?

Reply:
Aha this is the continuation on comment above! We could add Gen maybe. But it is in Generator module so I thought that would be enough?